test/cypress/integration/consumer-tools/find-a-housing-counselor/find-a-housing-counselor-helpers.cy.js

```diff
@@ -28,7 +28,7 @@ export class FindAHousingCounselor {
 
   searchZipCode(zipCode) {
     cy.get('#hud-hca-api-query').type(zipCode);
-    cy.get('[data-cy=btn-fahc-submit]').click();
+    cy.get('#hud_search_container button[type="submit"]').click();
```

Comment:
Whoops! I wiped this out in https://github.com/cfpb/consumerfinance.gov/commit/28944283f9ce51a52ed387e72ace4ef527d25c7f when the search was migrated to the standard search input. I guess we'd have to modify the search input template to allow a custom data-cy attribute, so I think your fix here is fine for now.